pkg/kubelet/config/defaults.go

```diff
@@ -26,4 +26,5 @@ const (
 	DefaultKubeletContainersDirName          = "containers"
 	DefaultKubeletPluginContainersDirName    = "plugin-containers"
 	DefaultKubeletPodResourcesDirName        = "pod-resources"
+	DefaultKubeletPluginsDirSELinuxLabel     = "system_u:object_r:container_file_t:s0"
```

Comment:
nit: This is called a default, but is it configurable in any way? Should it just be `KubeletPluignsDirSELinuxLabel`?